packages/expo-cli/src/commands/start/TerminalUI.js

```diff
@@ -142,9 +142,15 @@ export const startAsync = async projectDir => {
         return;
       }
       case 'w': {
+        const { exp } = await ProjectUtils.readConfigJsonAsync(projectDir);
+        const isWebConfigured = exp.platforms.includes('web');
+        if (!isWebConfigured) {
+          log(Web.getWebSetupLogs());
```

Comment:
Could you move this check and logging the error message to [`Web.openProjectAsync`](https://github.com/expo/expo-cli/blob/9137987338f5a4dbc372bbfba6f2653df2d7a3d5/packages/xdl/src/Web.js#L7)? Then the check would automatically also run when people run `expo start --web` ([here](https://github.com/expo/expo-cli/blob/9137987338f5a4dbc372bbfba6f2653df2d7a3d5/packages/expo-cli/src/urlOpts.js#L89-L91)).